cilium/cmd/bpf_mountfs_show.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/cilium/cilium/pkg/mountinfo"
+	"github.com/spf13/cobra"
+)
+
+// bpfmountfsShowCmd represents the bpf file system show command
+var bpfmountfsShowCmd = &cobra.Command{
+	Use:     "show",
+	Short:   "Show bpf filesystem mount details",
+	Example: "cilium bpf fs show",
+	Run: func(cmd *cobra.Command, args []string) {
+		getbpfmountFS(cmd, args)
+	},
+}
+
+func init() {
+	bpfmountFSCmd.AddCommand(bpfmountfsShowCmd)
+}
+
+func getbpfmountFS(cmd *cobra.Command, args []string) {
+	var mountPoint, mountfsType string
+	var mountfsStatus bool
+	mountdetails, err := mountinfo.GetMountInfo()
+	if err != nil {
+		Fatalf("Unable to find the mount %s", err)
+	}
+	for _, mountInfo := range mountdetails {
+		if mountInfo.FilesystemType == "bpf" {
+			mountPoint = mountInfo.MountPoint
+			mountfsType = mountInfo.FilesystemType
+			mountfsStatus = true
+			break
+		}
+	}
+	w := tabwriter.NewWriter(os.Stdout, 2, 0, 3, ' ', 0)
+	fmt.Fprintf(w, "Filesystem Mounted state:\t%t\n", mountfsStatus)
+	fmt.Fprintf(w, "Filesystem Mount point:\t%s\n", mountPoint)
+	fmt.Fprintf(w, "Filesystem Type:\t%s\n", mountfsType)
```

Comment:
Even if there's just one line, I think it makes sense to print these in columns to be consistent with the other CLI formats.

Reply:
Ok will check it out.

Reply:
Done